.github/workflows/secret_scan.yml

```diff
@@ -2,21 +2,21 @@
 name: Secret Scanning Template
 
 on:
-    pull_request:
+  pull_request:
 
 jobs:
-    secret_scanning:
-        runs-on: ubuntu-latest
-        steps:
-            - uses: getsentry/trufflehog_reusable_workflow_test@main
-              id: call_trufflehog
-              continue-on-error: true
-            - uses: mshick/add-pr-comment@b8f338c590a895d50bcbfa6c5859251edc8952fc
-              if: steps.call_trufflehog.outcome != 'success'
-              with:
-                 message: |
-                   🚨🚨🚨 Secret found 🚨🚨🚨
-                   Please review your commit
-            - name: Fail workflow if secret detected
-              if: steps.call_trufflehog.outcome != 'success'
-              run: exit 1
\ No newline at end of file
+  secret_scanning:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: getsentry/action-secret-scanning@v1
+        id: call_trufflehog
+        continue-on-error: true
+      - uses: mshick/add-pr-comment@b8f338c590a895d50bcbfa6c5859251edc8952fc
+        if: steps.call_trufflehog.outcome != 'success'
+        with:
+          message: |
+            🚨🚨🚨 Secret found 🚨🚨🚨
+            Please review your commit
```

Comment:
Is this something we really need? Our PRs already get a lot of automated comments.
I think as long as the check ends up failed with the red X, it's sufficient.

This action hasn't been updated since Feb 1st and has some known vulnerable dependencies.

Reply:
Definitely not necessary, was thinking that this can provide more visibility but also see how it can be noisy.

I'll take a look at SAC workflows too; we are using this action in there as well.

Reply:
Removed it!